tests/functions.py

```diff
@@ -50,27 +38,35 @@ def recurse(directory, odds):
                 return "No annexkey in file: " + full_path
 
             # Check for authentication
-            if msg["status"] == "error" and \
-                "unable to access" not in msg["message"].lower() and \
-                "not available" not in msg["message"].lower():
+            if (msg["status"] == "error" and
+                    "unable to access" not in msg["message"].lower() and
+                    "not available" not in msg["message"].lower()):
                 return "No authentication setup for file: " + full_path
 
     return "All good"
 
 
 def examine(dataset):
 
-    # Number of files to test in each dataset
-    num_files = 100
-
     # Root directory can be projects or investigators
     root_dir = "projects" if dataset in listdir("projects") else "investigators"
+    full_dir = join(root_dir, dataset)
+
+    # Check if dats.json and README.md are present in root of dataset
+    # if "dats.json" not in listdir(full_dir):
```

Comment:
Hi @joeyzhou98 do you validate at all whether the dats model file is a a valid json-ld or if there are errors, apart from checking if the file exists? I have some work on that that we can add in case. Thanks

Reply:
This is out of the scope of this current PR. As discussed with @PapillonMcGill, this will be added to Travis once this PR is merged.